internal/controller/serverclaim_controller.go

```diff
@@ -180,7 +182,7 @@ func (r *ServerClaimReconciler) reconcile(ctx context.Context, log logr.Logger,
 	log.V(1).Info("Ensured PowerState for Server", "Server", server.Name)
 
 	log.V(1).Info("Reconciled server claim")
-	return ctrl.Result{}, nil
+	return ctrl.Result{RequeueAfter: r.ResyncInterval}, nil
```

Comment:
Here as well. The desired power state is transported to the `Server` object. And there we already do the polling and ensuring the correct state. Not sure we should enforce a requeueing here on the `ServerClaim` level.

Reply:
Removed this requeue, this should then be covered by above comment.